core/src/main/scala/pureconfig/package.scala

```diff
@@ -19,6 +19,9 @@ import pureconfig.error._
 
 package object pureconfig {
 
+  // an alias for a `Derivation` of a `ConfigReader`
+  private type DerivedConfigReader[A] = Derivation[ConfigReader[A]]
```

Comment:
An optional comment that you can ignore if you want: this alias doesn't buy you anything but makes the code more obscure to read. I would just delete it and use `Derivation[ConfigReader[A]]` everywhere.